src/test/suite/StartIntegration.test.ts

```diff
@@ -194,9 +194,21 @@ async function createFile(showQuickpickStub: sinon.SinonStub<any[], any>,
 }
 
 function checkTelemetry(telemetrySpy: sinon.SinonSpy<any[], any>, languageExtension: string) {
-	expect(telemetrySpy.calledOnce, `telemetry expected to be called once but was called ${telemetrySpy.callCount}`).true;
+	expect(telemetrySpy.calledOnce, `telemetry expected to be called once but was called ${telemetrySpy.callCount}.\n${getTelemetryCallsContent(telemetrySpy)}`).true;
```

Comment:
To finish the sentence, I would add ` times` after the `${telemetrySpy.callCount}`